gulpfile.js

```diff
@@ -25,7 +25,8 @@ const createElectronServer = (env, args = []) => {
     spawnOpt: {
       env: Object.assign({}, process.env, env),
       args,
-    }
+    },
+    port: process.env.ELECTRON_PORT
```

Comment:
@daniloprates this is not what we are aiming for :(
It is my fault as I did not explain the task good enough...
It is the `WALLET_PORT` that we need and not `ELECTRON_PORT`.
Backend will be running on different ports for different networks. Until now we always used port `8090` to contact it. This has to be changed so that we use this variable: https://github.com/input-output-hk/daedalus/blob/devops-922-testnet-parallel/source/common/environment.js#L21 in all of our API calls (for Cardano only!). This means that we need to replace hardcoded `8090` port from all of the places and use the one from environment file.
This is one of the places where we need to do that: https://github.com/input-output-hk/daedalus/blob/devops-922-testnet-parallel/source/renderer/app/api/ada/adaTxFee.js#L22
(there are many more)
Thanks for taking care of this! :)

Reply:
Sorry for the misunderstanding 👎Now it's done 👍